### PR TITLE
Created New Schema And Specifications For Phone Audio Input Plugin 

### PR DESCRIPTION
**Pull request by this-Aditya**
## Description

This PR introduces a new schema for the `PhoneAudioInput` plugin, which collects uncompressed audio data.

## Changes Made

- Added Schema `phone_audio_input.avsc`
- Created specification for this schema

### CODE REVIEW

File: specifications/passive/android_phone-1.0.0.yml [hunk 1]
```diff
@@ -156,3 +156,11 @@ data:
     value_schema: .passive.phone.PhoneUserInteraction
     sample_rate:
       dynamic: true
+  # Phone Audio Input
+  - type: PHONE_AUDIO_INPUT
```
Comment (yatharthranjan) [inline specifications/passive/android_phone-1.0.0.yml:160]: I think this should be in a separate specifications file

Comment (yatharthranjan) [reply specifications/passive/android_phone-1.0.0.yml:160]: Yeah i am not sure (they are likely closely related to the phone). For audio input, it is pretty separate (similar to opensmile audio).

File: specifications/passive/android_phone_audio-1.0.0.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+vendor: ANDROID
+model: PHONE
+version: 1.0.0
+data:
+  # Phone Audio Input
+- type: PHONE_AUDIO_INPUT
+  app_provider: .phone.PhoneAudioInputProvider
+  processing_state: RADAR
+  topic: android_phone_audio_input
+  value_schema: .passive.phone.PhoneAudioInput
```
Comment (yatharthranjan) [inline specifications/passive/android_phone_audio-1.0.0.yml:11]: use the correct schema path (.passive.audio.PhoneAudioInput)

Comment (this-Aditya) [reply specifications/passive/android_phone_audio-1.0.0.yml:11]: Sorry!! my mistake.

**Review by yatharthranjan — APPROVED** [review]
LGTM, thanks

**Pull request merged**